src/checks/json-schema-errors.ts

```diff
@@ -1,6 +1,9 @@
 import Ajv, { ErrorObject } from "ajv";
+import jsonata from "jsonata";
+import addFormats from "ajv-formats";
+import common from "../schemas/common.json";
 import paths from "../schemas/paths.json";
-import jsonata from "../schemas/jsonata.json";
+import jsonataSchema from "../schemas/jsonata.json";
```

Comment:
Would it make sense to suffix all the schemas by `Schemas`?

Reply:
yes, I'll make that change.

I only did this one to avid conflicting w/ the library name import